packages/mui-material/src/Accordion/Accordion.js

```diff
@@ -262,12 +293,13 @@ Accordion.propTypes /* remove-proptypes */ = {
   /**
    * The component used for the transition.
    * [Follow this guide](/material-ui/transitions/#transitioncomponent-prop) to learn more about the requirements for this component.
-   * @default Collapse
+   * @deprecated Use `slots.transition` instead. This prop will be removed in v7.
```

Comment:
Not sure about adding "This prop will be removed in v7," but I think it provides clarity and urgency.

Reply:
Agree.